content/scout/integrations/ci/gha.md

```diff
@@ -63,17 +63,15 @@ steps:
   - name: Setup Docker buildx
     uses: docker/setup-buildx-action@v3
 
-  # Login against a Docker registry except on PR
-  # https://github.com/docker/login-action
-  - name: Log into registry ${{ env.REGISTRY }}
+  # Authenticate to the container registry
+  - name: Authenticate to registry ${{ env.REGISTRY }}
     uses: docker/login-action@v3
     with:
       registry: ${{ env.REGISTRY }}
-      username: ${{ secrets.DOCKER_USER }}
-      password: ${{ secrets.DOCKER_PAT }}
+      username: ${{ secrets.REGISTRY_USER }}
+      password: ${{ secrets.REGISTRY_TOKEN }}
```

Comment:
Not entirely sure this is the right change.
Docker scout needs the user (or GHA) to be authenticated against Docker Hub. Even if using a different container registry.
That's because we are using this as a way to authenticate queries to the backends.

Reply:
Yeah, I have the docker auth "again" on line 129: https://github.com/docker/docs/pull/18591/files#diff-4db6a3c8953b0fdb42c83b399304761ab99cb43b0e51a31af836177978257b5cR129

This is a bit awkward but I wanted this example to show what it would look like in case you need to auth twice - once with e.g. ecr and again with Docker.

Make sense?

Reply:
Ho yeah, sounds good, I missed this block (don't know how 🤔 )